### PR TITLE
Fixing lint warnings due to PR 4081

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Fixing lint warnings due to PR 4081

https://jenkins.catrob.at/job/Catroid/job/PR-4081/7/analysis/

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s gitflow workflow
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/config/lint-baseline.xml [hunk 1]
```diff
@@ -23,90 +23,35 @@
   -->
 <issues format="5" by="lint 3.5.2" client="gradle" variant="catroidDebug" version="3.5.2">
 
+    <issue
```
Comment (juliajulie95) [inline catroid/config/lint-baseline.xml:26]: We need to look into this, this doesn't seem like something we should ignore

File: catroid/config/lint-baseline.xml [hunk 2]
```diff
@@ -23,90 +23,35 @@
...
+        id="MissingPermission"
+        message="Missing permissions required by intent Intent.ACTION_CLOSE_SYSTEM_DIALOGS: android.permission.BROADCAST_CLOSE_SYSTEM_DIALOGS"
+        errorLine1="   sendBroadcast(it);"
+        errorLine2="   ~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/java/org/catrobat/catroid/utils/notifications/StatusBarNotificationManager.java"
+            line="265"
+            column="4"/>
+    </issue>
+
```
Comment (juliajulie95) [inline catroid/config/lint-baseline.xml:36]: Could you please delete these lines and test if it still works?

**Review by juliajulie95 — CHANGES_REQUESTED** [review]

File: catroid/config/lint-baseline.xml (hunk 2 above)
Comment (CodeChamp-SS) [reply catroid/config/lint-baseline.xml:36]: sure

**Review by juliajulie95 — APPROVED** [review]

**Comment by wslany** [comment]
Thanks.

**Pull request merged**